### PR TITLE
Wizard onepage

### PR DESCRIPTION
**Pull request by calebthompson**

### CODE REVIEW

File: public/templates/partials/wizard/multiple_choice.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
   >
     <input
       type="radio"
-      name="answer"
+      ng-name="option.question_id"
```
Comment (masonforest) [inline public/templates/partials/wizard/multiple_choice.html]: can we make this `question-{option.question_id}`?

File: app/assets/javascripts/wizards/all_questions_controller.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      vm.answers = {};
+      vm.projectId = $stateParams.projectId;
+      vm.questions = questions;
+      vm.submitAnswers = submitAnswers;
+      vm.tags = [];
+      vm.$state = $state;
+
+      function submitAnswers() {
+        vm.questions.forEach(function(question) {
+          var answer = vm.answers[question.id];
```
Comment (masonforest) [inline app/assets/javascripts/wizards/all_questions_controller.js]: Where does `vm.answers` come in? Just curious.

File: public/templates/partials/wizard/multiple_choice.html (hunk 1 above)
Comment (stackus) [inline public/templates/partials/wizard/multiple_choice.html]: Out of curiosity are you using a module that provides `ng-name` or did you mean to use `ng-model`?

File: app/assets/javascripts/wizards/all_questions_controller.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    /**@ngInject*/
+    .controller('WizardAllQuestionsController', function($stateParams, $state, questions) {
+      var vm = this;
+
+      vm.answers = {};
+      vm.projectId = $stateParams.projectId;
+      vm.questions = questions;
+      vm.submitAnswers = submitAnswers;
+      vm.tags = [];
+      vm.$state = $state;
```
Comment (stackus) [inline app/assets/javascripts/wizards/all_questions_controller.js]: Looks like you can leave $state off the viewmodel. I mean unless you really enjoy typing `vm.` to access it, it's within scope below and can be accessed directly without issue.

File: public/templates/partials/wizard/multiple_choice.html (hunk 1 above)
Comment (calebthompson) [inline public/templates/partials/wizard/multiple_choice.html]: Neither. What's weird is that I was seeing behavior as if each radio had the same name and so all clicks reset other radios. That's done now, and I get a random name for each radio.

Fun...

File: app/assets/javascripts/wizards/all_questions_controller.js (hunk 2 above)
Comment (calebthompson) [inline app/assets/javascripts/wizards/all_questions_controller.js]: It's populated [here](https://github.com/projectjellyfish/api/pull/655/files#diff-f5c696ad452296daaea8997d093bd6f0R7)

File: app/assets/javascripts/wizards/all_questions_controller.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+(function() {
+  'use strict';
+
+  angular.module('broker.wizard')
+    /**@ngInject*/
+    .controller('WizardAllQuestionsController', function($stateParams, $state, questions) {
+      var vm = this;
+
+      vm.answers = {};
+      vm.projectId = $stateParams.projectId;
```
Comment (masonforest) [inline app/assets/javascripts/wizards/all_questions_controller.js]: Probably don't need to put this on the `vm` either.

**Comment by masonforest** [comment]
:+1:

**Pull request merged**